### DISCUSSION TITLE
Rationale for not supporting write permission for Binary data type

### DISCUSSION THREAD

**Discussion by sudhamani-hcl** [edgexfoundry/community#33] [General]
Hi Team,
Currently I am implementing command support for REST device service.
As part of this I wanted to support code for all datatypes. During this,
I have come across that, Write support is not there for Binary data type device resources from the reference - https://github.com/edgexfoundry/go-mod-core-contracts/blob/main/dtos/deviceprofile.go , Line - 105
May I know the rationale behind this?
Thanks,
Sudhamani.

**Comment by iain-anderson** [comment]
Hi, it's just a matter of priority - it's a feature that we'd like to have at some point. There's an issue for support in the device sdk here: https://github.com/edgexfoundry/device-sdk-go/issues/488

**Comment by sudhamani-hcl** [comment]
Thanks Iain for the response. I am closing this issue.
